### DISCUSSION TITLE
Cron timezone correcting

### DISCUSSION THREAD

**Discussion by colgan1** [semaphoreui/semaphore#1960] [Ideas]
Good day!
I see an incorrect shown time in jobs running on schedule. If check database then date in jobs running  manually add timezone UTC, jobs running on cron - local system timezone. So display time shown incorrectly.
Can u please fix this? Or any manually fix?

**Comment by bodzio4749** [comment]
I had the opposite problem but fixed it by setting TZ="......"  in the docker-compose.  This fixed the job scheduler time. 

To display date in 24 hour (military) format use the "date -R"   command. Environment variable LANG defines all kinds of defaults related to your local. Time format is one of them. LC_TIME is more specific. You may want to investigate that.
